chart/compass/charts/gateway/values.yaml

```diff
@@ -15,3 +15,6 @@ gateway:
   auditlog:
     enabled: false
     authMode: "basic"
+
+metrics:
+  port: 3001
```

Comment:
I assume that the ports here should somehow correspond to the default values described in `main.go`'s `config` struct.

So isn't this supposed to be `port: 3003` as described in `main.go` at line 39 `MetricsAddress  string envconfig:"default=127.0.0.1:3003"` ? 

Port `3001` seems to be assigned already to the director proxy service (`DirectorOrigin  string envconfig:"default=http://127.0.0.1:3001"`).

Reply:
It is using port 3003 by default, as to not clash with the director local port, which is 3001. Anyway it is overwritten by the values.yaml in the chart.